.travis.yml

```diff
@@ -1,25 +1,30 @@
 language: php
 
-matrix:
-    include:
-        - php: hhvm
-        - php: 5.3
-        - php: 5.4
-        - php: 5.5
-        - php: 5.6
-        - php: nightly
-    allow_failures:
-        - php: hhvm
-        - php: nightly
-    fast_finish: true
-
 sudo: false
 
 cache:
   directories:
     - $HOME/.composer/cache
 
-install: composer install -n
+matrix:
+  include:
+    - php: 5.3
+    - php: 5.4
+    - php: 5.5
+    - php: 5.6
+    - php: 5.6
+    - php: 5.6
+      env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' 
```

Comment:
I don't get what you mean :/ .

I think adding the PHP 5.3-lowest build is interesting to check if the dependencies do work with PHP5.3 but I'm not sure it's very useful as we already have the classic PHP5.3 and the PHP5.6-lowest. It just adds a build for not much value.